test/test.js

```diff
@@ -258,4 +258,16 @@ QUnit.module('@yelm', () => {
 		assert.equal(testInstance.props.foo, 'New Prop Value');
 		assert.equal(divComponent.innerText, 'test New Prop Value');
 	});
+
+	QUnit.test('ObserveArray.map() shouldn\'t convert jsx to proxies', (assert) => {
```

Comment:
It's not map specific, actually. Jsx should never be observed. I think this test is to specific.

Reply:
What are some other scenarios where JSX would normally be converted? I can add them to the test.

Reply:
```
observe( <div/> )
```

should not proxy.